cmd/apiserver-boot/boot/run.go

```diff
@@ -63,20 +63,13 @@ func RunRun(cmd *cobra.Command, args []string) {
 		defer etcdCmd.Process.Kill()
 	}
 
-	time.Sleep(time.Second * 2)
```

Comment:
These are actually needed right now, because each process needs to wait for the previous one to fully start.  This could be fixed with polling the health endpoints of the processes.